hack/jenkins/e2e-runner.sh

```diff
@@ -116,7 +116,7 @@ function get_latest_trusty_image() {
       image="$(gsutil cat "gs://${image_project}/image-indices/latest-test-image-${image_index}")" && break || sleep 1
```

Comment:
Partially because of [this](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/faster_reviews.md#4-dont-rename-reformat-comment-etc-in-the-same-pr), but mainly because even though it seems a one line change, I need to do a lot of testing to make sure it works, and also, I have a follow up change that will use image family instead of these text files to locate the latest GCI image. So I'd rather keep things isolated to make review easier.